src/dist/RunLauncher.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Terminal=true
+Exec=sh -c 'bash "$(dirname "%k")"/bin/TerasologyLauncher'
+Name=Launcher
+StartupNotify=true
+GenericName[en_GB]=This is the Terasology Launcher for Linux
```

Comment:
GenericName should state what you would generally call an application that does what this specific application offers (i.e. Firefox is a "Web Browser"). ([source](https://wiki.archlinux.org/index.php/Desktop_entries#Key_definition)) Change this to something like "Game Launcher".